ci/run.sh

```diff
@@ -87,22 +84,70 @@ case $TRAVIS_OS_NAME in
         ;;
 esac
 
-if [ $TRAVIS_OS_NAME = osx ]; then
-    path=target/${1}/debug/deps/libcompiler_builtins-*.rlib
-else
+if [ -d /target ]; then
     path=/target/${1}/debug/deps/libcompiler_builtins-*.rlib
+else
+    path=target/${1}/debug/deps/libcompiler_builtins-*.rlib
 fi
 
+# Look out for duplicated symbols when we include the compiler-rt (C) implementation
 for rlib in $(echo $path); do
-    stdout=$($PREFIX$NM -g --defined-only $rlib)
+    set +x
+    stdout=$($PREFIX$NM -g --defined-only $rlib 2>&1)
 
-    # NOTE On i586, It's normal that the get_pc_thunk symbol appears several times so ignore it
+    # NOTE On i586, It's normal that the get_pc_thunk symbol appears several
+    # times so ignore it
+    #
+    # FIXME(#167) - we shouldn't ignore `__builtin_cl` style symbols here.
     set +e
-    echo "$stdout" | sort | uniq -d | grep -v __x86.get_pc_thunk | grep 'T __'
+    echo "$stdout" | \
+      sort | \
+      uniq -d | \
+      grep -v __x86.get_pc_thunk | \
+      grep -v __builtin_cl | \
+      grep 'T __'
+
+    if test $? = 0; then
+        exit 1
+    fi
+    set -ex
+done
+
+rm -f $path
+
+# Verification of the `intrinsics` program doesn't work on thumb targets right
+# now.
+case $1 in
```

Comment:
Ah so right now it actually has more to do with `panic_unwind`. Linking the example on MinGW required the panic runtime for all the requisite symbols (hard to get around otherwise manually). I figured that the this wouldn't get us much more coverage than the ARM targets already get, but do you think it's worth the `#[cfg]` to only link `panic_unwind` on Windows and get this working again for Thumb?